android/src/main/java/com/pspdfkit/react/events/PdfViewAnnotationChangedEvent.java

```diff
@@ -57,6 +57,7 @@ public void dispatch(RCTEventEmitter rctEventEmitter) {
                 annotationMap = new HashMap<>();
                 annotationMap.put("name", annotation.getName());
                 annotationMap.put("creatorName", annotation.getCreator());
```

Comment:
On iOS, we are not including the creator's name when an annotation is deleted:

https://github.com/PSPDFKit/react-native/blob/1ff022010e57db2ae1e7b169547693c84b551e9f/ios/RCTPSPDFKit/Converters/RCTConvert%2BPSPDFAnnotation.m#L27

The change would be pretty trivial:

```diff
- [annotationsJSON addObject:@{@"uuid" : annotation.uuid, @"name" : annotation.name ?: [NSNull null]}];
+ [annotationsJSON addObject:@{@"uuid" : annotation.uuid, @"name" : annotation.name ?: [NSNull null], @"creatorName" : annotation.user ?: [NSNull null]}];
```

@davidschreiber can you please make this change in this PR? Otherwise, I can do it myself in the PR or a followup PR.

Pinging @nickwinder, as I am not sure what Windows does when an annotation is deleted.